nilearn/maskers/tests/test_surface_maps_masker.py

```diff
@@ -265,181 +260,3 @@ def test_surface_maps_masker_sample_mask_to_fit_transform(
     )
     # we remove two samples via sample_mask so we should have 3 samples
     assert signals.shape == (3, masker.n_elements_)
-
-
-# ------------------ Tests for reporting ------------------
```

Comment:
@man-shu 
FYI: I am centralizing all the test for masker reports in reporting/tests/test_html_report.py